src/style.css

```
@@ -1 +1,78 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body,
+header,
+img,
+nav,
+ul,
+li,
+a {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  width: 100%;
+  padding: 0 50px;
+
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav-list {
+  display: flex;
+  flex-direction: row;
+
+  list-style: none;
+}
+
+.nav-link {
+  position: relative;
+
+  margin-right: 20px;
```

```suggestion
  position: relative;
  margin-right: 20px;
```
in case you have only 2 styles there is no need in space